broccoli/packages.js

```diff
@@ -16,24 +16,10 @@ const { stripIndent } = require('common-tags');
 const toES5 = require('./to-es5');
 
 module.exports.routerES = function _routerES() {
-  return new Rollup(findLib('router_js', 'lib'), {
+  return new Rollup(findLib('router_js'), {
     rollup: {
       external: ['route-recognizer', 'rsvp'],
-      entry: 'router.js',
-      plugins: [{
-        transform(code, id) {
```

Comment:
These overrides are not needed any longer, the upstream repo has corrected its invalid internals...